util/build_configs/cray-internal/README.md

```diff
@@ -6,7 +6,7 @@ one of the setenv scripts in this subdir, setenv-example-3 from the parent direc
 or a new setenv script derived from one of those. See `chapel_build.bash` for illustration.
 
 The Chapel RPM file produced by these scripts is intended to be fully compatible
-with current generation Cray-XC/XE supercomputers and system-management tools.
+with current generation Cray-XC supercomputers and system-management tools.
```

Comment:
no hyphen in "Cray XC"

Reply:
These scripts are internal enough that I'm not sure it's worth correcting.